lib/index.js

```diff
@@ -41,6 +41,21 @@ Intercom.ensure(function(msg){
   }
 });
 
+Intercom.prototype.initialize = function(){
+  this.userAgent = 'Segment.io/1.0.0';
+  // Intercom asked us to split some users on the old version to help with load
+  if (this.settings.isV1Enabled) {
```

Comment:
Can we name this to be more descriptive? Something like `bulkApiEnabled` so that anyone reading the settings knows exactly what it does.